Dockerfile

```diff
@@ -22,6 +22,13 @@ RUN curl --fail https://sh.rustup.rs -sSf \
     rustup default "${RUST_VERSION}-x86_64-unknown-linux-gnu" && \
     rustup target add wasm32-unknown-unknown
 
+# Install yq. We use it to read YAML file(s).
```

Comment:
We'd also need to remark that in the special instructions for Apple M1 users.

Reply:
Since the 1-proposal instructions are still being drafted, I'm going to leave the M1 section in the main README alone for now.

In lieu of such changes, I have added a TODO to the draft instructions to remind us to make such changes to the main README later, once README has been converted to 1-proposal.